Automattic/wp-calypso
# Performance Profiler: add image previews to insights list

Closes https://github.com/Automattic/dotcom-forge/issues/8955

## Proposed Changes

* add screenshots to insights list

## Why are these changes being made?


https://github.com/Automattic/dotcom-forge/issues/8955 

## Testing Instructions



~~Apply the diff D161557-code to your sandbox and sandbox `public-api`~~ deployed
* Go to `/speed-test` and start a new test
  * You can also use these test URLs:
    * `/speed-test-tool?url=https://www.atliq.com/&hash=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6NzY4NH0.iVJiilTIiXIgSEaE8eq0PikeQl0lHAR_ZRVRWS2gJc8&tab=mobile`
    * `/speed-test-tool?url=https://wordpress.com/&hash=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6NzY4N30.niHEAudz33_rtd8DZxiGUxRPmDMbqgV9L4kJZnWq0Oc&tab=desktop`
* When the test completes, scroll down to the insights section and open an insight that contains nodes
* Check that the thumbnails for the insights are rendered correctly
* Check that when clicking on the thumbnails, the overlay image is displayed correctly

| Before | After |
|--------|--------|
| <img width="1025" alt="image" src="https://github.com/user-attachments/assets/3d3d705d-b06a-45c6-86f5-1e8319d6055e"> | <img width="1029" alt="image" src="https://github.com/user-attachments/assets/71856723-86c8-4bcd-973a-3a252b674a19"> | 

## Pre-merge Checklist
